.circleci/config.yml

```diff
@@ -402,6 +402,17 @@ jobs:
             RUST_BACKTRACE=1 $CARGO $CARGOFLAGS test \
               --features='vanilla' \
               --no-default-features
+  compile-benches:
+    executor: audit-executor
+    description: Compile (but don't run) the benchmarks, to insulate against bit rot
+    steps:
+      - build_setup
+      - restore_cargo_package_cache
+      - run:
+          name: Run bench --no-run
+          command: |
+            RUST_BACKTRACE=1 $CARGO $CARGOFLAGS x bench \
+              --no-run
```

Comment:
One more nit - we need `-- --no-run`

Reply:
yep :facepalm:

Reply:
Actually, we need pass-through arguments on x's `bench` command, it seems. :tired_face: